pkg/node/node.go

```diff
@@ -1534,93 +1534,63 @@ func (s *Service) readFCPortsFilterFile() ([]string, error) {
 
 func (s *Service) setupHost(initiators []string, client gopowerstore.Client, arrayIP, arrayID string) error {
 	log.Infof("setting up host on %s", arrayIP)
-	defer log.Infof("finished setting up host on %s", arrayIP)
 
 	if s.nodeID == "" {
 		return fmt.Errorf("nodeID not set")
 	}
 
 	if s.useNVME[arrayID] {
-		// Check for duplicate hostnqn uuids
 		s.checkForDuplicateUUIDs()
 	}
 
 	reqInitiators := s.buildInitiatorsArray(initiators, arrayID)
-
-	var host *gopowerstore.Host
-	updateCHAP := false
-
-	h, err := client.GetHostByName(context.Background(), s.nodeID)
-	if err == nil {
-		err := s.updateHost(context.Background(), initiators, client, h, arrayID)
-		if err != nil {
-			return err
-		}
-		if s.opts.EnableCHAP && len(h.Initiators) > 0 && (h.Initiators[0].ChapSingleUsername == "" || h.Initiators[0].ChapSingleUsername == "admin") {
-			log.Debug("CHAP was enabled earlier, modifying credentials")
-			err := s.modifyHostInitiators(context.Background(), h.ID, client, nil, nil, initiators, arrayID)
-			if err != nil {
-				return fmt.Errorf("can't modify initiators CHAP credentials %s", err.Error())
-			}
-		}
-
-		s.initialized = true
-		return nil
-	}
+	var existingHost *gopowerstore.Host
 
 	hosts, err := client.GetHosts(context.Background())
 	if err != nil {
-		log.Error(err.Error())
```

Comment:
Any reason to not log the exact error here?

Reply:
This is fixed.